src/main/windows/PickerWindowController.js

```diff
@@ -1,62 +1,89 @@
+const MouseCaptureHandler = require("../resources/MouseCaptureHandler");
 
 /**
  * PickerWindowController Class
  *
  * Handles the magnification picker window
  */
 class PickerWindowController {
-
   /**
    * @constructor
    * @param {WindowManager} wm the WindowManager instance for the app
    */
   constructor(wm) {
     this.window = null;
+    this._MouseCaptureHandler = new MouseCaptureHandler(wm, this);
     this.createWindow(wm);
     this.isVisible = false;
   }
 
+  /**
+   * Get the picker size of the mouse capture handler
+   *
+   * @memberof PickerWindowController
+   */
+  get pickerSize() {
+    return this._MouseCaptureHandler._PickerSize;

```

Comment:
Seems hacky, improve this as modifying private vars outside of class should not be allowed